tests/acceptance/features/bootstrap/SpacesContext.php

```diff
@@ -3122,4 +3178,38 @@ public function asUserFileInsideSpaceShouldContainAPropertyWithValue(
 			$expectedValue
 		);
 	}
+
+	/**
+	 * @Given /^the content of file "([^"]*)" inside space "([^"]*)" for user "([^"]*)" should be "([^"]*)"$/
+	 *
+	 * @param $fileSource
+	 * @param $spaceName
+	 * @param $user
+	 * @param $expectedContent
+	 *
+	 * @return void
+	 * @throws GuzzleException
+	 */
+	public function theContentOfFileInsideSpaceForUserShouldBe($fileSource, $spaceName, $user, $expectedContent) {
+		$user = $this->featureContext->getActualUsername($user);
+		$space = $this->getSpaceByName($user, $spaceName);
+		$fullUrl = $space["root"]["webDavUrl"] . '/' . ltrim($fileSource, "/");
+		$this->featureContext->setResponse(
+			HttpRequestHelper::get(
+				$fullUrl,
+				$this->featureContext->getStepLineRef(),
+				$user,
+				$this->featureContext->getPasswordForUser($user),
+			)
+		);
+		$this->featureContext->theHttpStatusCodeShouldBe(200);
+		$actualContent = (string) $this->featureContext->getResponse()->getBody();
+		Assert::assertEquals(
+			$expectedContent,
+			$actualContent,
+			__METHOD__ . " Downloaded content mis-match for resource at '$fileSource' inside space '$spaceName'" .
+			"\n- Expected content: $expectedContent" .
+			"\n+ Actual content: $actualContent"
+		);
+	}
```

Comment:
We have a step `And for user "Alice" the content of the file "/testshare.txt" of the space "Personal" should be "testshare content"` to check content. can be reused if it is same.

Reply:
missed that, reused the existing step.